chainstate/chainstate.go

```diff
@@ -9,24 +9,44 @@ import (
 	"time"
 )
 
+type HexFormatFlag byte
```

Comment:
Minor - lint error:
`exported: exported type HexFormatFlag should have comment or be unexported (revive)go-golangci-lint`